src/Pages/PeoplePage.tsx

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from "react";
+import { IPage, PersonWithId } from "../Interfaces/Person";
+import PeopleServices from "../Services/People";
+import CreatePerson from "../Components/People/CreatePerson";
+import { toast } from "react-toastify";
+import AddButton from "../Components/AddButton";
+import Person from "../Components/People/Person";
+
+const PeoplePage: React.FC<IPage> = ({ cookie, currentRole }) => {
+  const [peopleList, setPeopleList] = useState<PersonWithId[]>([]);
+  const [isCreating, setIsCreating] = useState(false);
+
+  useEffect(() => {
+    PeopleServices.getAllPeople()
+      .then((res) => {
+        if (!res) return;
```

Comment:
Why would that happen?

Reply:
if there are no people theres nothing to return

Reply:
Empty array?